analytics-core/src/main/java/com/segment/analytics/ValueMap.java

```diff
@@ -51,6 +51,10 @@ public ValueMap() {
     delegate = new LinkedHashMap<>();
   }
 
+  public ValueMap(int initialCapacity) {
```

Comment:
Since the other constructor is public, it makes sense for this one to be
public as well.

On Wed, Jul 1, 2015, 11:10 Prateek Srivastava <notifications@github.com>
wrote:

> In analytics-core/src/main/java/com/segment/analytics/ValueMap.java
> <https://github.com/segmentio/analytics-android/pull/311#discussion_r33706685>
> :
>
> > @@ -51,6 +51,10 @@ public ValueMap() {
> >      delegate = new LinkedHashMap<>();
> >    }
> >
> > +  public ValueMap(int initialCapacity) {
>
> Is there any value in making this public over protected or
> package-private? Callers wouldn't really create instances of this directly.
> Those who really need the int overload, can create their own pre-sized maps
> and use the map constructor.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/segmentio/analytics-android/pull/311/files#r33706685>.
>